internal/connect/command_plugin_install.go

```diff
@@ -58,10 +62,19 @@ func (c *pluginCommand) newInstallCommand() *cobra.Command {
 
 	cobra.CheckErr(cmd.MarkFlagDirname("plugin-directory"))
 
+	c := &pluginInstallCommand{
+		CLICommand: pcmd.NewAnonymousCLICommand(cmd, prerunner),
+	}
+	cmd.RunE = c.install
+
 	return cmd
 }
 
-func (c *pluginCommand) install(cmd *cobra.Command, args []string) error {
+func (c *pluginInstallCommand) install(cmd *cobra.Command, args []string) error {
+	if runtime.GOOS == "windows" {
+		return fmt.Errorf("this command is not available on Windows")
+	}
```

Comment:
I guess this is better than hiding the command outright on Windows. Any other commands that could take advantage of this pattern? The new `confluent local` commands maybe?

Reply:
Hmm, maybe we should.

I don't think there's anything actually preventing windows users from using the local commands, but it is kind of awkward to allow users to demo something that isn't supported on their platform.

Reply:
Oh wait, it should be supported since we're using Docker. Never mind!